.github/workflows/tests.yml

```diff
@@ -11,6 +11,32 @@ on:
       - 'docs/**'
 
 jobs:
+  windows:
+    runs-on: windows-latest
+    strategy:
+      matrix:
+        python-version:
+          - '3.10'
```

Comment:
Do we only run Windows with 3.10?
Do we want to expand to 3.8 and 3.9 as there is more capacity with GitHub Actions that we had with Jenkins?

Reply:
> Do we only run Windows with 3.10?

Yes.

> Do we want to expand to 3.8 and 3.9 as there is more capacity with GitHub Actions that we had with Jenkins?

I don't see much value in doing this. I don't remember any Windows-specific regression on older Python versions, and we have [daily builds](https://djangoci.com/job/django-windows/) that will catch them anyway. Let's don't waste global resources :deciduous_tree: :evergreen_tree: